### DISCUSSION TITLE
ERROR:root:Failed to validate prompt for output 9:

### DISCUSSION THREAD

**Discussion by scottonanski** [comfyanonymous/ComfyUI#2308] [General]
I tried to run the default "Queue Prompt" and the following error was bounced back at me;

```
ERROR:root:Failed to validate prompt for output 9:
ERROR:root:* CheckpointLoaderSimple 4:
ERROR:root:  - Value not in list: ckpt_name: 'v1-5-pruned-emaonly.ckpt' not in []
ERROR:root:Output will be ignored
invalid prompt: {'type': 'prompt_outputs_failed_validation', 'message': 'Prompt outputs failed validation', 'details': '', 'extra_info': {}}
```

I'm new to running AI locally, so I really don't have a clue of what I'm doing.  Needless to say, I'm a little stuck.

That being said, I'm going to assume I don't have 'checkpoints' installed? I think that would be as good a place to start as any.

Here's a screenshot of what I'm looking at;

![image](https://github.com/comfyanonymous/ComfyUI/assets/110937649/a672a52e-66f1-4f0f-a500-4b14dfde2a17)

So, my question is where do I get the 'checkpoint' necessary to get up and running?

I'll check the docs again. Maybe I missed something.

**Comment by ltdrdata** [comment]
It seems there is a lack of knowledge about Stable Diffusion. In that case, I recommend watching this video for more insight.

https://youtu.be/zxjGLa960_I?feature=shared

I could simply guide you on where to download a single checkpoint, but that wouldn't be meaningful.

**Reply by amitkhadse81** [reply]
i m going to watch the video now i am also stuck same as him

**Comment by scottonanski** [comment]
Thanks. I figured it out.

**Reply by amitkhadse81** [reply]
hey how did you figured it out please let me know, i am also stuck.

**Comment by mulsewm** [comment]
how did you figured it out..any tips?

**Reply by ltdrdata** [reply]
That means you don't have any checkpoint model files.
Checkpoint file should be placed in `ComfyUI/models/checkpoints`.
And then refresh browser.
